src/tasks/label_projection/methods/scanvi_scarches/config.vsh.yaml

```diff
@@ -58,7 +58,7 @@ platforms:
     # Add custom dependencies here
```

Comment:
possibly also change the image for this task to `ghcr.io/openproblems-bio/base_pytorch_nvidia:1.0.3`